ksp/src/main/kotlin/com/livefront/sealedenum/internal/ksp/SealedEnumProcessor.kt

```diff
@@ -98,7 +98,6 @@ internal class SealedEnumProcessor(
         resolver: Resolver,
         sealedClassCompanionObjectKSClass: KSClassDeclaration
     ): SealedEnumFileSpec? {
-
```

Comment:
Was this required by the change in detekt?

Reply:
Yes, this change was enforced and auto-corrected by detekt.